widgetssdk/src/main/java/com/glia/widgets/callvisualizer/EndScreenSharingView.kt

```diff
@@ -109,10 +101,9 @@ internal class EndScreenSharingView(
         binding.root.applyLayerTheme(theme.background)
     }
 
-    private fun applyRuntimeTheme(theme: UiTheme?) {
-        if (theme == null) {
-            Logger.d(TAG, "UiTheme is null!")
-            return
+    private fun applyRuntimeTheme(theme: UiTheme) {
+        theme.iconEndScreenShare?.let {
+            binding.endSharingButton.icon = ResourcesCompat.getDrawable(context.resources, it, null)
```

Comment:
We were not applying style to the End Screen Sharing icon?

Reply:
we were, but just before calling this function.